CefSharp/CefDragOperationsMask.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright © 2010-2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Drag operation mask types
+    /// </summary>
+    [Flags]
+    public enum CefDragOperationsMask
```

Comment:
There's already a `CefDragOperationsMask` enum see https://github.com/cefsharp/CefSharp/blob/cef/2171/CefSharp/DragOperationsMask.cs

Just a note most enums in `Cef` are `uint`, whilst you can cast between the two, the value for `Every` is different being `uint.MaxValue`